### ISSUE TITLE
Windower 4 Will Not Open PlayOnline

### ISSUE THREAD

**Issue by SochenCave** [Windower/Issues#889]
I tried to launch Play Online through Windower and when you click the <-- key it says launching PlayOnline. I tried restarting my computer and redownloading the app it still doesnt work.

**Comment by z16** [comment]
What happens? Does it just disappear, without anything happening? Does it hang indefinitely? Does it give an error message? If it's the latter, in the Windower folder there should be a `crash.log` file. Can you post the contents of that file here?

Also, what Windower version are you using? You can find out by right-clicking Windower and going to *Properties* -> *Details*.

**Comment by SochenCave** [comment]
It hangs indefinitely on launching playonline. It never loads playonline in otherwords. Launcher 4.36328.2701... Hook:4.3.0.6 is everything i think

**Comment by akihitoh** [comment]
I have a similar problem with JP client. Launcher is showing "Launching PlayOnline". I can see that pol instance is launched in task viewer and does not show up any window. Thanks.

<img width="685" alt="2017-04-30 10 57 22" src="https://cloud.githubusercontent.com/assets/19888051/25560613/1f8bbb8a-2d94-11e7-99b7-2629f889a13c.png">
<img width="407" alt="2017-04-30 10 56 31" src="https://cloud.githubusercontent.com/assets/19888051/25560612/1f8a50d8-2d94-11e7-992d-1a55a388d7a4.png">

**Comment by z16** [comment]
Are you using a 32 bit Windows 10?

**Comment by zadelia** [comment]
the same problem happen to me

**Comment by oiton** [comment]
Sorry for not to mention my environment. 64bit Windows 10 for my case.

**Comment by Aznjason02** [comment]
I am having the exact same issue. Just hanging at launching PlayOnline.  Windows 7 32 bit here.

**Comment by Byrth** [comment]
@oiton are you using XiLoader?

**Comment by oiton** [comment]
@Byrth Thank you for commenting. I do not use any additional program for ff11 except windower 4. I have tried no firewall and lowest UAC level. But, no luck. Is there anything what I should try to investigate the problem?

**Comment by Selindrile** [comment]
Just wanted to say a friend of mine is having this same issue as well with windows 7 home premium SP1, just hangs up on "Launching Playonline" forever, he can launch Playonline fine without Windower.

Tried both the normal and Dev versions of Windower on his computer over Teamviewer, if you need any more information let me know.

**Comment by pxk01501** [comment]
I have same problem on Win7 32bit edtion.
Windower is displaying "Launching PlayOnline", but Pol does not appear.
It is in process list of TaskManager, but does not come out and doing something with using max cpu usage.
Manualy I killed the process of pol.exe and Windower stopped and created some messages in crash.log.
(Windower did not create messages while pol.exe was alive, was Windower  waiting for pol.exe?).

Conditions.

MyPC:Windows 7 Profesional SP1 32bit
Windower.exe:4.3.6328.2814
Hook :4.3.0.6
Crash.log contens after manualy stoped pol.exe:

System.ComponentModel.Win32Exception (0x80004005): access denied.
   at Windower.Core.Injector.Suspend()
   at Windower.Core.Injector.GetInstructionPointer()
   at Windower.Core.Injector.RunProcessInitialization()
   at Windower.Core.Injector.Inject(String dllPath)
   at Launcher.PlayOnlineLauncher.Launch(String hook, Profile profile, Boolean debug, TimeSpan timeout)
   at Launcher.Tasks.LaunchTask.Run()
   at Launcher.Tasks.Task.Run(Core core)
   at Launcher.Core.Run()
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
-

**Comment by Sesono** [comment]
The same thing happens to me on Win 8.1 32bit (no crash dump) 

I noticed something strange:
When i start vanilla POL the hook itself seems to be in place and working just fine, even though windower didnt load pol it still works in FFXI - but just that, no plugins/addons.

I tried to verify this and rebooted, then i started vanilla POL and the look of the game was horrible i was also unable to alt-tab without crashing (lost full screen message in POL).

So, even though it doesnt "bring up" POL it still works somehow if i start it prior of starting vanilla POL to log in.

**Comment by pxk01501** [comment]
I found old version of 4.3, called 4.3.632626.35073.
You can get from http://www.ffera.com/forums/viewtopic.php?f=2&t=3977&p=28759#p28759 .
This is stable.

**Comment by RichardNesbitt** [comment]
I tried pxk01501's suggestion and it didn't work.
I'm running 64-bit Windows 10. Fresh installed everything yesterday. Windower crashes when trying to launch POL.

**Comment by GurXI** [comment]
I Don't have anything new to add, I could not get a crash log, works with POL's , I tried to use other log in profiles on my PC did no good. I wiped my folder and re-installed and no change. Only thing I noticed is some times i would have multiple copies of POL running. i let my widower sit for over 10 minutes and it never gave me a crash log :(  ill try again  soon.

![windoer 1 8](https://cloud.githubusercontent.com/assets/28249988/25575328/675aacf8-2e13-11e7-9e84-5bf6cad531b8.JPG)

![windower 1 8](https://cloud.githubusercontent.com/assets/28249988/25575329/6b279ed6-2e13-11e7-8b1a-ccf51ae3fa8e.JPG)

![windower1 3](https://cloud.githubusercontent.com/assets/28249988/25575331/6f27dcb2-2e13-11e7-86fe-2032e081b4cc.JPG)
![windower1 5](https://cloud.githubusercontent.com/assets/28249988/25575342/85c90af4-2e13-11e7-93b3-f3629ec33fce.JPG)
![windower1 7](https://cloud.githubusercontent.com/assets/28249988/25575343/87a11998-2e13-11e7-8e88-32803c6e68d7.JPG)
![windower1 2](https://cloud.githubusercontent.com/assets/28249988/25575358/b703bed4-2e13-11e7-8184-be1c53813f26.JPG)
![windoer 1 9](https://cloud.githubusercontent.com/assets/28249988/25575360/bc46fa96-2e13-11e7-8760-e8585db42988.JPG)

**Comment by zaxtiss** [comment]
ahh i'm having the same issue ;; sorry i didnt notice there was a issue going for it already ;_;

**Comment by GurXI** [comment]
i was able to force it to crash when loading but don't know if this will show you anything your after it's all gebberish to me it crashed when just trying to load and I closed out Process Pol.exe this is a clean install 
Windows 7 pro SP1
AMD A8-3850 
Ram 3.25
official copy (dont know if that matters)


`````````````````````````````````````````````````````````````````````
[ 4.3.6328.2702 ] 2017-05-02 05:13:10.1203602 UTC
---------------------------------------------------------------------
System.ComponentModel.Win32Exception (0x80004005): Access is denied
   at Windower.Core.Injector.Suspend()
   at Windower.Core.Injector.GetInstructionPointer()
   at Windower.Core.Injector.RunProcessInitialization()
   at Windower.Core.Injector.Inject(String dllPath)
   at Launcher.PlayOnlineLauncher.Launch(String hook, Profile profile, Boolean debug, TimeSpan timeout)
   at Launcher.Tasks.LaunchTask.Run()
   at Launcher.Tasks.Task.Run(Core core)
   at Launcher.Core.Run()
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
`````````````````````````````````````````````````````````````````````

**Comment by BonusKun** [comment]
This same issue has been happening to me as well. I'm running this game thru Windows Vista and it's just hanging there with the same exact messages. Thanks for working on this by the way. I know you guys are probably swamped irl.

**Comment by alphaONE2** [comment]
Could you yous please try [this build](https://www.dropbox.com/s/n6g8afyh0vypdeo/windower.exe?dl=0), and let me know if it fixes anything?

**Comment by Selindrile** [comment]
Another friend having the same problem, Windows 10 Professional x64, Creator's Update, normal windower, dev version, and the build you linked, all same issue, crashing at the launching playonline screen, not hanging up.

**Comment by alphaONE2** [comment]
@Selindrile: This issue is about the launcher hanging indefinitely at the "Launching PlayOnline" stage. Crashing at launch is a separate issue. Please open a new issue and provide a crash log or crash dump if possible.

**Comment by Selindrile** [comment]
Sorry, I thought these issues were all related, I'll just have this one make a github account, he's more tech savvy than the other.

**Comment by GurXI** [comment]
It appears the same to me.  I tried to run a program with windows to capture the crash log (live) but i could not figure out how to make it run. (4.3.0.6) Launcher 4.3.6331.38835 

ill try that safe mode with netwroking here in a few minutes if i can get it going

I take screen shots a lot because a picture is worth 1000 words, sorry i have not got a chance to start my PC in safe mode with netwroking (forgot what F-key to press and it bugged me)  if you want any specific into feel free to let me know (don't know what i can provide but i can) 
![hook 4 3 0 6](https://cloud.githubusercontent.com/assets/28249988/25650032/516cfc9a-2f97-11e7-8b92-c23bce855c0e.JPG)

**Comment by GurXI** [comment]
Well good thing I have an iPad (FYI I hate(yes hate) apple). I started my pc in safe mode with networking and it had the same problem, at least that answered my question about it. The bad news is now it won't start networking without safe mode hahaha, thats my issue though. So  I don't know how much helps but here it is my safe mode with networking can't get on line but the loader is still not getting past the same point.

**Comment by BonusKun** [comment]
@alphaONE2, I went ahead and tried to run the file you posted. This time it attempts to run PoL but it crashes now instead of just hanging there. I just get the message "Playonline Viewer has stopped working" tab.

**Comment by pxk01501** [comment]
The module of the URL I showed was taken from those who maintain the client of the private FFXI server.

The URL was instructed to configure Windows Firewall so that updating will not be started when Windower is launched.
If you do not keep this, you will also be returned to the original troubled version of Windower.

I tried going according to this instruction. By replacing it with the old module, setting FireWall so that it can not be used by the network from Windower, and starting Windower, PlayOnLine appears on the screen, login is possible, and the previous AddOn works.

However, if I minimize the screen and return it will crash.
Also, since Windows key handling is different from 4.2, it is not completely same as before.

I guess that the cause of the problem is different between 32 bit and 64 bit OS.
I feel that in 32bit, PlayOnLine does not appear on the screen due to WOW problem, and in 64bit  PlayOnline is crashing with more pure programatic mistake.

I also want to contribute to everyone, but there is only research and forecast (speculation). My environment can not do anything because it does not output a crash dump at 32 bit.

Either way, I do not know what will happen if SqureEnix upgrades FFXI as it is.

We can only pray to the development team.
Good luck, the development team!

**Comment by GurXI** [comment]
Well my Windows 10 64 bit works(dual boot), I cant get my 32 bit back up and going, sorry I wont be able to provide any more info to this, When/if i get my win 7 re installed in time to help I will tho. Good Luck

**Comment by Travis959** [comment]
Just wanted to post in case, but if you are using Ashita and the dx8 to dx9 mod, you need to move those two files out of the PlayOnline folder when using Windower otherwise you will have this issue as well.

**Comment by BonusKun** [comment]
Just wanted to update that I've been seeing a few updates installing off and on so I know you guys have been hard at work trying to resolve whatever issues have been coming up. My Play Online crash tab comes up now every time I try and run the software and then it's still has the "Launching Play Online" still hanging in the background. 

Anyways I just wanted to thank you guys again for the work you've been doing.

**Comment by Byrth** [comment]
Are you on -dev or -live?

**Comment by BonusKun** [comment]
I'm currently on the live version. I tried to install and run the -dev a few days ago and it simply crashed PoL also.

**Comment by Byrth** [comment]
Two more questions:
* You are attempting to log in to a normal server and not using XiLoader, right?
* What Operating System are you using?

**Comment by BonusKun** [comment]
1st Question: Correct, I'm trying to log into the normal servers. I hadn't bothered to use any private servers nor have I used XiLoader. Just the normal Windower Software. I tried running it on -dev and then on -live and originally it was only hanging but but it instantly brings up the "Play Online Viewer has stopped working" Tab. A moment later, Windower itself crashes as well.

2nd Question: Windows Vista. Yeah yeah I know old ass OS!
